packages/api-derive/test/treasury/bounties.spec.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright 2017-2020 @polkadot/api authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { bounties } from '@polkadot/api-derive/treasury/bounties';
```

Comment:
The helpers are fine in test/ but prefer having the actual spec alongside the implementation.